### PR TITLE
Implement oclif

### LINKED ISSUE #22: Implement Oclif
- @oclif/command
- @oclif/test
- @oclif/plugin-help
- w/ first command and test

### PR DESCRIPTION
**Pull request by Joshua-Phelps**
closes #22 

- installed @oclif/command and @oclif/test
- added first dummy command and test

Reasons for using oclif:
- works well with typescript
- supports multi-commands
- Auto-parsing args and flags
- Config support
- Test framework
- Auto documenting

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,5 +45,12 @@
     "nyc": "^15.1.0",
     "ts-node": "^9.1.1",
     "typescript": "^4.0.3"
+  },
+  "dependencies": {
+    "@oclif/command": "^1.8.0",
+    "@oclif/test": "^1.2.8"
```
Comment (noelbundick) [inline package.json:51]: Can we move this to devDependencies?

**Review by noelbundick — APPROVED** [review]
approved w/ comment

File: package.json (hunk 1 above)
Comment (Joshua-Phelps) [reply package.json]: Yes

**Review by noelbundick — APPROVED** [review]
lgtm

**Pull request merged**